src/assets/toolkit/styles/utils/size.css

```diff
@@ -17,3 +17,7 @@
     max-height: 50vh !important;
   }
 }
+
+.u-widthFull {
```

Comment:
Ha, yeah. I just (finally) submitted an issue to SUITCSS for adding a utility for just `width: 100%` and that was the class name I suggested. I _didn't_ do it here because I thought it might be misleading if it didn't have all the responsive options like the other size utilities, but I guess that's silly. I'll change it up.